### ISSUE TITLE
Guild run queue stuck in a loop when unable to resolve dependency

### ISSUE THREAD

**Issue by samedii** [guildai/guildai#151]
```
ERROR: [queue] 2020-03-22 17:04:24 5d4a8a6130d1492a828c1b5f50b0e2da failed with exit code 1
INFO: [queue] 2020-03-22 17:04:24 Starting staged run 5d4a8a6130d1492a828c1b5f50b0e2da
Resolving input dependency
Skipping file:input because it's already resolved
Resolving model dependency
guild: run failed because a dependency was not met: could not resolve 'operation:train|retrain' in model
resource: no suitable run for train|retrain
ERROR: [queue] 2020-03-22 17:04:24 5d4a8a6130d1492a828c1b5f50b0e2da failed with exit code 1
INFO: [queue] 2020-03-22 17:04:24 Starting staged run 5d4a8a6130d1492a828c1b5f50b0e2da
Resolving input dependency
Skipping file:input because it's already resolved
Resolving model dependency
guild: run failed because a dependency was not met: could not resolve 'operation:train|retrain' in model
resource: no suitable run for train|retrain
ERROR: [queue] 2020-03-22 17:04:25 5d4a8a6130d1492a828c1b5f50b0e2da failed with exit code 1
INFO: [queue] 2020-03-22 17:04:25 Starting staged run 5d4a8a6130d1492a828c1b5f50b0e2da

```

**Comment by gar1t** [comment]
Thanks for the report! This is fixed and master and will be available in the next release.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report!
